src/ui/InteractiveBook/InteractiveBookFront.tsx

```diff
@@ -1,21 +1,10 @@
 import { Image } from '@chakra-ui/react';
 interface BookFrontTypes {
   src: string;
```

Comment:
p5;

관련된 타입은 Pick 해서 사용해봐도 좋을 것 같아요!

```suggestion
  src: Pick<APIBook, "url">;
```

Reply:
이와 관련된 부분들 점진적으로 바꿔나가야겠습니다 굳굳~~~